cpp/src/reductions/compound.cuh

```diff
@@ -53,35 +57,39 @@ std::unique_ptr<scalar> compound_reduction(column_view const& col,
 {
   auto const valid_count = col.size() - col.null_count();
 
+  // All null input produces all null output
+  if (valid_count == 0 ||
+      // Don't care about ddof argument of mean. For variance and
+      // standard deviation must have more valid entries than ddof.
```

Comment:
We could add more compound reductions here. Can we check "is var or std" rather than "not mean"?